### ISSUE TITLE
Validation of 834 file

### ISSUE THREAD

**Issue by ramanja21** [BerryWorksSoftware/edireader#20]
Does Edireader validate input 834 file?
If yes, which schema it uses?

**Comment by canabrook** [comment]
Yes, the EDIReader framework provides a number of value-added modules on top of the core EDIReader parser, including a validation module. It is driven from an EdiModel object which represents the various compliance checking "rules" for a given transaction type such as the 834. The EdiModel can be built directly and efficiently from an XSD licensed separately from X12. However, the EdiModel is fully decoupled from any particular schema, making it possible to build the model from another XSD or other expression of the validation rules.

**Comment by ramanja21** [comment]
Thank you for quick reply.
Is Validation module under GNU General Public License v3.0?  If yes how can we integrate into the EDIreader?
If its not open source/free license, then how can we get validation module and use in the EDIreader?

**Comment by canabrook** [comment]
The validator module is not open source. Contact me at ***@***.***
and we can discuss licensing info.
